### PR TITLE
Align naming in Channel cluster XML with the spec.

### PR DESCRIPTION
**Pull request by bzbarsky-apple**
REVIEW NOTE: Only the first commit contains manual changes. The rest is code-generated.

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml [hunk 1]
```diff
@@ -24,54 +24,54 @@ limitations under the License.
...
-      <arg name="count" type="INT16U"/>
+      <arg name="Count" type="INT16U"/>
     </command>
 
     <command source="server" code="0x01" name="ChangeChannelResponse" optional="true">
       <description>Upon receipt, this SHALL display the active status of the input list on screen.</description>
-      <arg name="status" type="ChannelStatusEnum"/>
-      <arg name="data"   type="CHAR_STRING" optional="true"/>
+      <arg name="Status" type="ChannelStatusEnum"/>
+      <arg name="Data"   type="CHAR_STRING" optional="true"/>
```
Comment (andy31415) [inline src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml:50]: octet string?

File: src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml [hunk 2]
```diff
@@ -24,54 +24,54 @@ limitations under the License.
...
       <description>Change the channel on the media player to the channel case-insensitive exact matching the value passed as an argument. </description>
-      <arg name="match" type="CHAR_STRING"/>
+      <arg name="Match" type="CHAR_STRING"/>
     </command>
 
     <command source="client" code="0x02" name="ChangeChannelByNumber" optional="false">
       <description>Change the channel on the media plaeyer to the channel with the given Number in the ChannelList attribute.</description>
-      <arg name="majorNumber" type="INT16U"/>
-      <arg name="minorNumber" type="INT16U"/>
+      <arg name="MajorNumber" type="INT16U"/>
```
Comment (andy31415) [inline src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml:38]: oddly enough, my copy of the spec has these starting with lowercase (even though that seems inconsistent with all other structs I have seen)

File: src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml [hunk 3]
```diff
@@ -24,54 +24,54 @@ limitations under the License.
...
 
-  <struct name="ChannelInfo">
+  <struct name="ChannelInfoStruct">
     <cluster code="0x0504"/>
-    <item name="majorNumber"       type="INT16U"/>
-    <item name="minorNumber"       type="INT16U"/>
-    <item name="name"              type="CHAR_STRING" optional="true" length="32"/>
-    <item name="callSign"          type="CHAR_STRING" optional="true" length="32"/>
-    <item name="affiliateCallSign" type="CHAR_STRING" optional="true" length="32"/>
+    <item name="MajorNumber"       type="INT16U"/>
```
Comment (andy31415) [inline src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml:57]: same here: minor and major in the spec are lowercase, however all the rest are uppercase.

File: src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml [hunk 4]
```diff
@@ -24,54 +24,54 @@ limitations under the License.
...
+    <item name="MinorNumber"       type="INT16U"/>
+    <item name="Name"              type="CHAR_STRING" optional="true" length="32"/>
```
Comment (andy31415) [inline src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml:59]: I see no length constraint in my copy of the spec.
Is this something that ember requires?

File: src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml (hunk 3 above)
Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml:57]: Spec has been updated recently.

File: src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml (hunk 4 above)
Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml:59]: No, it's completely ignored here.  I can take it out.

**Review by jmartinez-silabs — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml (hunk 1 above)
Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml:50]: Filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6204

File: src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml (hunk 4 above)
Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml]: Removed the lengths.

**Review by jtung-apple — APPROVED** [review]

**Pull request merged**